app/models/maestrano/connector/rails/entity.rb

```diff
@@ -32,7 +32,7 @@ def create_external_entity(mapped_connec_entity, external_entity_name)
   def update_external_entity(mapped_connec_entity, external_id, external_entity_name)
     Maestrano::Connector::Rails::ConnectorLogger.log('info', @organization, "Sending update #{external_entity_name} (id=#{external_id}): #{mapped_connec_entity} to #{Maestrano::Connector::Rails::External.external_name}")
 
-    @external_client.update(external_entity_name, external_id, mapped_connec_entity)
+    @external_client.create(external_entity_name, mapped_connec_entity)
```

Comment:
won't you need to merge the `external_id` into the `mapped_connec_entity` first? Because the mapped_entity won't contain any id